### PR TITLE
Add promtool check of alerting rules

### PR DESCRIPTION
**Pull request by idavidmcdonald**
Also takes advantage of travis global environment variables
for hopefully more readable set up code in our travis file.

https://docs.travis-ci.com/user/environment-variables/#global-variables

### CODE REVIEW

**Comment by philandstuff** [comment]
@idavidmcdonald I pushed a modification - feel free to back it out if you disagree

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,24 @@
 env:
-  - TERRAFORM_VERSION=0.11.7 TERRAFORM_FILE_NAME=terraform_${TERRAFORM_VERSION}_linux_amd64.zip TERRAFORM_DOWNLOAD_URL=https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/${TERRAFORM_FILE_NAME}
+  global:
+    - TERRAFORM_VERSION=0.11.7
+    - TERRAFORM_FILE_NAME=terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+    - TERRAFORM_DOWNLOAD_URL=https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/${TERRAFORM_FILE_NAME}
+    - PROMETHEUS_VERSION=2.3.2
+    - PROMETHEUS_FILE_NAME=prometheus-${PROMETHEUS_VERSION}.linux-amd64
+    - PROMETHEUS_TAR_FILE_NAME=${PROMETHEUS_FILE_NAME}.tar.gz
+    - PROMETHEUS_DOWNLOAD_URL=https://github.com/prometheus/prometheus/releases/download/v${PROMETHEUS_VERSION}/${PROMETHEUS_TAR_FILE_NAME}
```
Comment (philandstuff) [inline .travis.yml]: is it worth trying to cache this stuff?

Comment (philandstuff) [reply .travis.yml]: @idavidmcdonald told me IRL that travis recommends only caching for built artefacts, not for downloaded binaries.  this rings a bell with me so I'm happy 👍  (i can't be bothered finding the reference though)

Comment (idavidmcdonald) [reply .travis.yml]: https://docs.travis-ci.com/user/caching/#things-not-to-cache

**Pull request merged**